testsuite/cluster-test/src/instance.rs

```diff
@@ -167,6 +168,22 @@ impl Instance {
         &self.peer_name
     }
 
+    pub fn peer_id(&self) -> Result<String> {
+        let re = Regex::new(r"val-(\d+)").unwrap();
```

Comment:
Regex compilation is quite expensive, most common pattern is to use `lazy::`:
https://docs.rs/regex/1.3.6/regex/#example-avoid-compiling-the-same-regex-in-a-loop

We already had problem in cluster test once when regex compilation was 90% of CPU time in some other case

Reply:
Done